src/components/APIquote.js

```diff
@@ -0,0 +1,46 @@
+import { useState, useEffect } from 'react';
+import './APIquote.css';
+
+const APIquote = () => {
+  const [data, setData] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, sethasError] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const url = 'https://api.api-ninjas.com/v1/quotes?category=life';
+      try {
+        setIsLoading(true);
+        const res = await fetch(url, {
+          method: 'GET',
+          headers: { 'X-Api-Key': 'rqBsI0z9wOGAqafKnCkkEA==KMMCPBuS2rX5v54e' },
+          contentType: 'application/json',
+        });
+        const response = await res.json();
+        setData(response);
+      } catch (error) {
+        sethasError(true);
+      }
+      setIsLoading(false);
+    };
+    fetchData();
+  }, [setData, setIsLoading]);
+
+  if (hasError) return <div>An error occured while loading data</div>;
+
+  if (isLoading) return <div>Loading...</div>;
+
+  return (
+    <ul>
+      {data.map((item) => (
+        <>
+          <p key={item.author}>{item.quote}</p>
+          <quote key={item.author}><i>{item.author}</i></quote>
```

Comment:
- [ ] @dazehinn great work in your app, however,am getting the errors below in your console due to the use of similar keys in lines 37 and 38 (you might encounter one key error due to nature of the API)
![image](https://user-images.githubusercontent.com/61408860/235318575-7ad122a9-f53f-4357-8bc8-3d001e65967f.png)

Reply:
@danmainah ,
please do you have any suggestions? The data from the API does not contain any unique values.